x-pack/plugins/enterprise_search/public/applications/app_search/components/result_field_value/result_field_value.scss

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.enterpriseSearchDataType {
+  &--number,
+  &--float {
+    color: $euiColorDanger;
```

Comment:
@constancecchen For sure, I like that. What do you and @daveyholler think about using a color and roboto font for dates as well? It's the one non-string data type that we don't have a special treatment for and I'm not sure why.

Reply:
You can see a date in the date_established field above. It gets the same treatment as strings, currently.

Reply:
Ya I was a little confused about dates as well but didn't comment since it was the same as before. I think it would be a great improvement to add a color & mono font to it now!

Reply:
I know this is being ported over from existing models, but I'm having a hard time not interpreting the colors as success/error right now.

With that said, and in an attempt to not throw wrenches around, I suggest utilizing a simple mono font for the date object and applying a `warning` if you _absolutely_ need to 🙂 but leaving it the standard color if you can. For the sake of clarity, using the blue/primary color will probably make it look more like a link.

Reply:
Oh... and we need to make sure we're using the text variants of these colors for accessibility/contrast. `$euiColorAccentText` instead of `$euiColorAccent` 
<img width="426" alt="image" src="https://user-images.githubusercontent.com/739960/99832141-f8e48c00-2b14-11eb-867b-483d34dd5391.png">

Reply:
Oh cool, thanks Davey. That is great to know.

Reply:
TIL!! Thanks Davey.

Question, could we just have dates use the accent or secondary color as well? It wouldn't be all that bad reusing the same color, right?

But yeah at least a mono font would help I think!

Reply:
Yeah we could totes reuse those colors. Probably secondary since it looks like it's only being used once.

Reply:
I just gave it a new font for now: https://github.com/elastic/kibana/pull/83564/commits/9ab27d8b15683c48b308e764da025e4f2d6e8aed

![screencapture-localhost-5601-app-enterprise-search-app-search-engines-national-parks-demo-documents-park-wind-cave-2020-11-20-13_54_57](https://user-images.githubusercontent.com/1427475/99845766-2b0ce280-2b43-11eb-8cbe-b9107b2642f1.png)